Kajabi/pine
# DSS-474: feat() - Add `error-message` to radio & checkbox

# Description

The checkbox and radio are missing the separate `error-message` prop.
This adds the `error-message` prop, documentation, and tests to match other components.

Fixes [DSS-474](https://kajabi.atlassian.net/browse/DSS-474)

## Type of change


# How Has This Been Tested?

1. Navigate to the checkbox and radio.
2. Verify error-message displays as expected. (see docs and invalid story)
3. Verify that error-message tests pass as expected. (`yarn nx run core:test`)


**Test Configuration**:

- Pine versions:
- OS:
- Browsers:
- Screen readers:
- Misc:

# Checklist:

If not applicable, leave options unchecked.



[DSS-474]: https://kajabi.atlassian.net/browse/DSS-474?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
